LibreTransmitter/LibreTransmitterManagerV3.swift

```diff
@@ -22,6 +22,8 @@ public final class LibreTransmitterManagerV3: CGMManager, LibreTransmitterDelega
 
     public let isOnboarded = true   // No distinction between created and onboarded
 
+    private var alertsUnitPreference = DisplayGlucosePreference(displayGlucoseUnit: .milligramsPerDeciliter)
```

Comment:
Just an initial default; Loop will call unitDidChange with the user's preferred units right after initialization.